src/features/chat/components/UserAvatar.tsx

```diff
@@ -0,0 +1,30 @@
+import React /*, { CSSProperties }*/ from 'react'
+import { ChatUser } from '../common'
+
+export interface ChatUserAvatarProps {
```

Comment:
Please use type instead of interface and add T

Reply:
changed